docs/Gemfile.lock

```diff
@@ -29,6 +29,7 @@ GEM
     ffi (1.16.3)
     forwardable-extended (2.6.0)
     gemoji (4.1.0)
+    google-protobuf (3.25.3-arm64-darwin)
```

Comment:
you've locked all this to ARM, undo, if you want to build use an AMD64/x86 .devcontainer

Reply:
not intentional. Will remove.